examples/server_readspeed.c

```diff
@@ -13,11 +13,20 @@
 #define UA_SERVER_H_
 #define UA_CONNECTION_H_
 #define UA_TYPES_GENERATED_H_
+#define UA_LOG_H_
+#define UA_SECURITYCONTEXT_H_
+#define UA_SECURITYPOLICY_H_
 #include "server/ua_services.h"
 #include "ua_types_encoding_binary.h"
 
 int main(int argc, char** argv) {
     UA_ServerConfig config = UA_ServerConfig_standard;
+
+    // Set desired security policies
+    UA_SecurityPolicy securityPolicies[] = { UA_SecurityPolicy_None };
```

Comment:
Currently, we use a static variable that contains the default serverconfig.
We could instead define function that populates and returns the default config.
That could lead to much cleaner code. Static assignment of large structures is a mess.

But when the default config contains allocated memory, we need the API for cleaning up as well.
This could become inevitable with the context attached to security policies.